resources/common/common.tf

```diff
@@ -148,6 +148,7 @@ data "terraform_remote_state" "etcd" {
         bucket = "${var.remote_state_bucket}"
         key = "etcd.tfstate"
         region = "${var.remote_state_region}"
+        role_arn = "${var.aws_account["role_arn"]}"
```

Comment:
Yeah, no function to set/unset attributes. These will have to be generated too. The goal of keeping things simple, declarative have to give-in to use cases. I will look in to it too.